config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml

```diff
@@ -151,3 +316,79 @@ periodics:
       hostPath:
         path: /sys/fs/cgroup
         type: Directory
+
+# presubmit jobs
+
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-kind
```

Comment:
this needs to be forked for old releases too

Reply:
ok, will add the other branches too.

Reply:
actually, is there an easy way to do that?
e.g. for a PR that is on `release-1.12` to call `/test pull-kuberentes-e2e-kind` to test this branch?